projects/storefrontlib/src/cms-components/misc/icon/icon.model.ts

```diff
@@ -10,6 +10,8 @@ export enum ICON_TYPES {
   ERROR = 'ERROR',
   WARNING = 'WARNING',
   SUCCESS = 'SUCCESS',
+  PLUS = 'plus',
```

Comment:
This should be `PLUS='PLUS'` and `MINUS='MiNUS'`. Then in `fontawesome-icon.config.ts` add `PLUS='plus'` and `MINUS='minus'`

Reply:
But with this the styling looks strange